logic/settings.ts

```diff
@@ -61,6 +63,7 @@ export const COUNT_TYPE_DESCRIPTIONS: { [countType: string]: string } = {
 	[CountType.Modified]:
 		"Date of last edit. (On folders: latest edit date of any note.)",
 	[CountType.FileSize]: "Total size on hard drive.",
+	[CountType.CustomKey]: "Key in the front matter block.",
```

Comment:
In common usage, I think "frontmatter" is usually one word.

```suggestion
	[CountType.CustomKey]: "Key in the frontmatter block.",
```